islands/src20/SRC20Header.tsx

```diff
@@ -28,35 +24,12 @@ export const SRC20Header = (
     setIsOpen2(open);
   };
 
-  const handleResultClick = (result: SearchResult) => {
-    globalThis.location.href = `/src20/${result.tick}`;
-  };
-
   return (
     <div className="tabs">
       <div class="flex flex-row justify-between items-center gap-3 w-full">
-        <div class="flex gap-6 tablet:gap-8 items-end">
-          <p
-            class={`cursor-pointer pb-1 tablet:pb-3 text-base tablet:text-2xl uppercase ${
-              selectedTab === "all"
-                ? "text-[#AA00FF] border-b-2 border-b-[#AA00FF] font-bold"
-                : "text-[#8800CC] font-light"
-            }`}
-            onClick={() => setTypeOption("src20", "all", true)}
-          >
-            All
-          </p>
-          <p
-            class={`cursor-pointer pb-1 tablet:pb-3 text-base tablet:text-2xl uppercase ${
-              selectedTab === "trending"
-                ? "text-[#AA00FF] border-b-2 border-b-[#AA00FF] font-bold"
-                : "text-[#8800CC] font-light"
-            }`}
-            onClick={() => setTypeOption("src20", "trending", true)}
-          >
-            Trending
-          </p>
-        </div>
+        <h1 className="text-4xl mobileLg:text-6xl purple-gradient1 font-black">
```

Comment:
are these theh only breakpoints used for the font size in figma ?

Reply:
let me check the figma again